compiler-rt/lib/mpk_untrusted/alloc_site_handler.h

```diff
@@ -38,31 +43,30 @@ namespace __mpk_untrusted {
  * pointer, then add them to the associated set of the newly generated AllocSite
  * to ensure that if a reallocated site is found to be unsafe, the original
  * allocation can also be marked unsafe.
- *
- * @note A note on thread safety: The only parameter that is changed at any
```

Comment:
I added the comment back with alterations, but it is also possible that it no longer applies as we are now dealing with copies instead of pointers to object. Thus, outside of modifying an AllocSite within the map in AllocSiteHandler (which currently no function does) there should never be two threads that have access to the same AllocSite. Each thread would have its own copy of AllocSite.